assets/stylesheets/navbar.css

```
@@ -0,0 +1,69 @@
+ul {
+	-webkit-user-select: none;
+    user-select: none;
+    position: fixed;
+    list-style-type: none;
+    padding: 0;
+    background-color: #333;
+    margin: 0;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 60px;
+}
+
+li {
+    float: left;
+}
+
+/* dont worry my css is bad too :) */
+
+li a {
+    display: block;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    color: azure;
+    font-weight: bolder;
+    font-size: 24px;
+    text-align: center;
+    padding: 14px 32px;
+    text-decoration: none;
+    border-radius: 8px;
+}
+
+li a:hover:not(.home):not(.active):not(li#github a) {
+    background-color: #1c8d75; /* Green for non-active links */
+}
+
+li a.home, .active {
+    background-color: #40c5d6;
+    color: azure;
+}
+
+li a.active:hover {
+    background-color: #40c5d6; /* Keeps cyan background on hover */
+    color: azure;
+}
+
+li#github a img {
+    width: 32px;
+    height: 32px;
+}
+
+li#github {
+    float: right;
+}
```

Temporary github button fix

```suggestion
li#github {
    float: right;
    a {
        /* (Navbar height - icon height) / 2 (left-right, top-bottom) */
        padding: calc((60px - 32px) / 2);
    }
}
```